tests/test_utils.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python3
```

Comment:
what's a "fixtures" folder... you mean "data" folder?

Reply:
One common practice I saw in python testing is to keep the data you need for input/output validation for testing in a folder called fixtures. If you have other suggestions for naming, we can go ahead with that.

Reply:
Ok. Interesting. That's new to me. We typically called it simply as data.. that way, the paths to json files look like .."tests/data/input.json" which reads very easily. 

Not important, either way is fine